app.js

```diff
@@ -18,7 +17,7 @@ const chalk = require('chalk');
 
 const processOne = require('./lib/process');
 const sitemap = require('./lib/sitemap');
-const dns = require('./lib/dns-cache');
+require('./lib/dns-cache');
```

Comment:
Still need the module ?

Reply:
Yes, the module patches node's dns module, so we just need to load it once.